releases/release-1.19/README.md

```diff
@@ -52,21 +52,23 @@ The 1.19 release cycle is proposed as follows:
 | **Call for [Exceptions][Exception]** | Lead | Mon, June 1 | week 8 | |
 | 1.19.0-beta.1 released | Branch Manager | Tue, June 2 | week 8 | |
 | Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon, June 8 | week 9 | |
-| Docs deadline - Open placeholder PRs | Docs Lead | Fri, June 12 | week 9 | |
 | 1.19.0-beta.2 released | Branch Manager | Tue, June 9 | week 9 | |
-| **Begin [Code Freeze]** (EOD PST) | Branch Manager | Thu, June 25 | week 11 | |
-| 1.19.0-rc.1 released | Branch Manager | Thu, June 25 | week 11 | |
-| release-1.19 branch created | Branch Manager | Thu, June 25 | week 11 | |
-| release-1.19 jobs created | Branch Manager | Thu, June 25 | week 11 | |
-| Docs deadline - PRs ready for review | Docs Lead | Mon, June 29 | week 12 | |
-| 1.19.0-rc.2 released | Branch Manager | Tue, July 7 | week 13 | |
-| Docs complete - All PRs reviewed and ready to merge | Docs Lead | Thu, July 9 | week 13 | |
+| Docs deadline - Open placeholder PRs | Docs Lead | Fri, June 12 | week 9 | |
```

Comment:
Hi @onlydole, Can the deadline be updated to `June 19th`?

Reply:
@savitharaghunathan - absolutely!